### PR TITLE
[ADD] Remove uniqueness constraint on bank account number

### PR DESCRIPTION
**Pull request by mclaeysb**
This functionallity [exists](https://github.com/coopiteasy/vertical-cooperative/blob/9.0/easy_my_coop/models/res_partner_bank.py#L5) in Easy My Coop - Vertical Cooperative. To be able to add it to Easy My Hub and other projects, we create a separate addon for it.

### CODE REVIEW

File: res_partner_bank_remove_uniqueness_constraint/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (houssine78) [inline res_partner_bank_remove_uniqueness_constraint/__init__.py]: to be compliant to PEP8 you should have a carriage return

File: res_partner_bank_remove_uniqueness_constraint/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2018 Open Architects Consulting SPRL.
+# Copyright 2018-Coop IT Easy SCRLfs (<http://www.coopiteasy.be>)
+# - Houssine BAKKALI - <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).#
```
Comment (houssine78) [inline res_partner_bank_remove_uniqueness_constraint/__manifest__.py:4]: add you name here ;)

File: res_partner_bank_remove_uniqueness_constraint/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import res_partner_bank_remove_uniqueness_constraint
```
Comment (houssine78) [inline res_partner_bank_remove_uniqueness_constraint/models/__init__.py]: add a line

File: res_partner_bank_remove_uniqueness_constraint/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "category": "Cooperative management",
+    "website": "www.coopiteasy.be",
+    "license": "AGPL-3",
+    "description": """
+    This module removes the SQL uniqueness constraint in the Bank Account model
+    res.partner.bank that requires every bank account to have a unique
+    (sanitized) account number.
+    """,
+    'data': [
+    ]
```
Comment (houssine78) [inline res_partner_bank_remove_uniqueness_constraint/__manifest__.py]: if data is empty you can get ride of it ;)

**Review by houssine78 — APPROVED** [review]
just take my comments into account

File: res_partner_bank_remove_uniqueness_constraint/__init__.py (hunk 1 above)
Comment (houssine78) [reply res_partner_bank_remove_uniqueness_constraint/__init__.py]: Rémy told be that you shoud see that with Robin as it's a PyCharm config

**Review by houssine78 — APPROVED** [review]

**Review by houssine78 — APPROVED** [review]

**Review by remytms — APPROVED** [review]

**Comment by remytms** [comment]
@mclaeysb Je comprends seulement maintenant que ce module est pour la version 11.0 et pas la 12.0.

Je pense que ce serait bien de porter ce module aussi vers la 12.0. :)

**Comment by remytms** [comment]
@houssine78 What do you think about porting this module also to 12.0 ?

**Comment by houssine78** [comment]
@houssine78 yes it could be nice. But as it works as it is, it's not a priority. We should handle the other taskes first.

**Comment by mclaeysb** [comment]
Is it up to me to merge this?

**Comment by houssine78** [comment]
@mclaeysb if it passed the test it can be merged

**Pull request merged**